charts/stable/nextcloud/templates/_hpb.tpl

```diff
@@ -73,8 +73,21 @@ command:
 
     {{- if .Values.imaginary.enabled }}
     echo  "Imaginary High Performance Previews enabled, enabling it on Nextcloud..."
+    php /var/www/html/occ config:system:set enabledPreviewProviders 0 --value='OC\Preview\PNG'  # This might need to be an array like https://github.com/nextcloud/server/blob/master/config/config.sample.php#L1183
+    php /var/www/html/occ config:system:set enabledPreviewProviders 1 --value='OC\Preview\JPEG'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 2 --value='OC\Preview\GIF'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 3 --value='OC\Preview\BMP'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 4 --value='OC\Preview\XBitmap'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 5 --value='OC\Preview\MP3'
     php /var/www/html/occ config:system:set enabledPreviewProviders 6 --value='OC\Preview\Imaginary'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 7 --value='OC\Preview\MarkDown'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 8 --value='OC\Preview\OpenDocument'
+    php /var/www/html/occ config:system:set enabledPreviewProviders 9 --value=> 'OC\Preview\TXT'
```

Comment:
```suggestion
    php /var/www/html/occ config:system:set enabledPreviewProviders 9 --value='OC\Preview\TXT'
    php /var/www/html/occ config:system:set enabledPreviewProviders 10 --value='OC\Preview\Krita'    
```
Those are by default on. But we can add them anyway
https://github.com/nextcloud/server/blob/74ebb72622301a912dcea0c5d9e194f6a5784d0a/config/config.sample.php#L1150-L1194

Reply:
Also should we add couple of checkboxes on the GUI to let the user easily enable the rest of the providers?
```
 * The following providers are disabled by default due to performance or privacy
 * concerns:
 *
 *  - OC\Preview\Illustrator
 *  - OC\Preview\HEIC
 *  - OC\Preview\Movie
 *  - OC\Preview\MSOffice2003
 *  - OC\Preview\MSOffice2007
 *  - OC\Preview\MSOfficeDoc
 *  - OC\Preview\PDF
 *  - OC\Preview\Photoshop
 *  - OC\Preview\Postscript
 *  - OC\Preview\StarOffice
 *  - OC\Preview\SVG
 *  - OC\Preview\TIFF
 *  - OC\Preview\Font
 ```

Reply:
> Those are by default on. But we can add them anyway https://github.com/nextcloud/server/blob/74ebb72622301a912dcea0c5d9e194f6a5784d0a/config/config.sample.php#L1150-L1194

Yeah our default was only 6 (Imaginary) since we only had that in our config, me and @ZasX did some tests and we need the defaults specified only with Imaginary


> Also should we add couple of checkboxes on the GUI to let the user easily enable the rest of the providers? I can do it if you agree
> 
> ```
>  * The following providers are disabled by default due to performance or privacy
>  * concerns:
>  *
>  *  - OC\Preview\Illustrator
>  *  - OC\Preview\HEIC
>  *  - OC\Preview\Movie
>  *  - OC\Preview\MSOffice2003
>  *  - OC\Preview\MSOffice2007
>  *  - OC\Preview\MSOfficeDoc
>  *  - OC\Preview\PDF
>  *  - OC\Preview\Photoshop
>  *  - OC\Preview\Postscript
>  *  - OC\Preview\StarOffice
>  *  - OC\Preview\SVG
>  *  - OC\Preview\TIFF
>  *  - OC\Preview\Font
> ```

I saw that option and didn't want to go too big too soon but I like the idea haha

Reply:
Now that I'm thinking.. we also have to `unset` when those are `false`